docs/mongocli/command/mongocli-atlas-networking-peering-create-azure.txt

```diff
@@ -16,9 +16,9 @@ Create a network peering connection between the Atlas VPC and your Azure VNet.
 
 Before you create an Azure network peering connection, complete the prerequisites listed here: https://www.mongodb.com/docs/atlas/reference/api/vpc-create-peering-connection/#prerequisites.
 		
-		The network peering create command checks if a VNet exists in the region you specify for your Atlas project. If one exists, this command creates the peering connection between that VNet and your VPC. If an Atlas VPC does not exist, this command creates one and creates a connection between it and your VNet.
+The network peering create command checks if a VNet exists in the region you specify for your Atlas project. If one exists, this command creates the peering connection between that VNet and your VPC. If an Atlas VPC does not exist, this command creates one and creates a connection between it and your VNet.
 		
-		To learn more about network peering connections, see https://www.mongodb.com/docs/atlas/security-vpc-peering/.
+To learn more about network peering connections, see https://www.mongodb.com/docs/atlas/security-vpc-peering/.
```

Comment:
**QUESTION**: Is it worth abstracting this out to an include file since it appears twice already? If it were more than twice, I'd strongly suggest this, but in this case I'd consider it a coin toss unless we suspect we might want to include this in other places later.

Reply:
We can't use rST includes in this repo unfortunately